features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/mobileapp/js/logic/recent-assets.js

```diff
@@ -38,7 +38,7 @@ $(function () {
             type: 'GET',
             success: function (response) {
 
-                var respObj=JSON.parse(response);
+                var respObj = response;
```

Comment:
removed redundant variable 'respObj' in commit 5fe5f85267ccc394f6b8fb0bee71b4c078d0f57f of PR #240